src/pages/form-validation/verify-on-submit/index.tsx

```diff
@@ -0,0 +1,243 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import {
+    AppBar,
+    Button,
+    CircularProgress,
+    Hidden,
+    IconButton,
+    InputProps,
+    Slide,
+    TextField,
+    Toolbar,
+    Typography,
+} from '@material-ui/core';
+import { Add, CheckCircle, Menu, Search } from '@material-ui/icons';
+import { makeStyles, Theme, useTheme } from '@material-ui/core/styles';
+import { useDispatch } from 'react-redux';
+import { TOGGLE_DRAWER } from '../../../redux/actions';
+import { EmptyState } from '@pxblue/react-components';
+
+type OnChangeHandler = InputProps['onChange'];
+
+const useStyles = makeStyles((theme: Theme) => ({
+    appbarRoot: {
+        padding: 0,
+    },
+    toolbarGutters: {
+        padding: '0 16px',
```

Comment:
can we change it to 

padding: `0 ${theme.spacing(4)}px`,

Reply:
Ok I've updated this spacing.